extensions/amp-access/0.1/amp-login-done.html

```diff
@@ -29,43 +74,51 @@
       display: none;
     }
 
-    .button-section {
-      text-align: center;
-    }
-
     #closeButton {
-      background: #4285f4;
-      color: #fff;
-      font-size: 14px;
-      font-weight: 500;
-      padding: 10px 20px;
-      border-radius: 2px;
-      box-shadow: 0 2px 5px 0 rgba(0,0,0,0.4);
-      outline: none;
-      border: none;
+      display: block;
+      color: #4285f4;
+      font-weight: 300;
+      text-decoration: none;
+      padding: 8px 0;
```

Comment:
could you sort the css rules  - alphabetically (-* would be at the bottom). Not sure if this is AMP style guide. we should make this our style guide if we don't already follow one.